doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -255,18 +287,57 @@ We are using a User OS based on `Clear Linux`_.
 
   .. code-block:: none
 
-     -s 3,virtio-blk,/root/clear-22780-kvm.img
-     -k /lib/modules/kernel/org.clearlinux.pk414-standard.4.14.47-44
+     -s 3,virtio-blk,/root/clear-24040-kvm.img
+     -k /lib/modules/kernel/org.clearlinux.pk414-standard.4.14.57-69
 
 Start the User OS (UOS)
 ***********************
 
 You are now all set to start the User OS (UOS)
 
-.. code-block:: none
+ .. code-block:: none
 
    sudo /usr/share/acrn/samples/nuc/launch_uos.sh
 
 **Congratulations**, you are now watching the User OS booting up!
 
+
+Enabling network sharing 
+************************
+
+After booting up SOS and UOS, enabling network sharing is must to test networking in UOS side. 
+enabling the tap and networking bridge in the SOS, you can create a script file to setup.
+below is an example for you reference:(verifed in Ubuntu 14.04 and 16.04 as the SOS) 
+ 
+ .. code-block:: none
+  
+    #!/bin/bash
```

Comment:
Is this script available on the NUC or in the GitHub repo, or will users need to create this themselves?

Reply:
user need to create